### DISCUSSION TITLE
Why functions being stripped out of list of objects in `arg()`

### DISCUSSION THREAD

**Discussion by ramiroaraujo** [johnlindquist/kit#1156] [Q&A]
I'm prompting the user with a list of transformation functions to perform on strings, and the functions inside the payload are being stripped out. I solved it with a Map, but just as a curiosity why being removed?

**Comment by johnlindquist** [comment]
@ramiroaraujo The payloads are serialized into JSON to send between the script process and the app process. 

So yeah, all handlers/fns/etc need to map to something valid in JSON in the payload if you want to invoke them.

Is there some specific behavior you're looking for I could support?
